### PR TITLE
Feature/no images found support

### LINKED ISSUE #14: No images found support
We have no indicator for when no images are found for a search. Ooops.gif

### PR DESCRIPTION
**Pull request by popthestack**
Displays "Could not find any gifs for that search term." when no results are found. It also prompts the user to enter a search term if they try searching for nothing.

Fixes #14.

### CODE REVIEW

File: src/app/search.service.ts [hunk 1]
```diff
@@ -4,38 +4,51 @@ import { Subject, Observer } from 'rxjs';
...
+                        }
+                    };
+
+                    return image;
+                }));
+
+                return results;
+            })
+            .catch(error => {
+                this.searchResults.next([]);
```
Comment (popthestack) [inline src/app/search.service.ts]: Not sure if this is the proper way to clear out an observable. It works, but I'm new to observables so I'm not sure.

Comment (adamweeks) [inline src/app/search.service.ts]: ¯\_(ツ)_/¯

**Comment by adamweeks** [comment]
Looks good to me. Ready to merge?

**Comment by popthestack** [comment]
Yep!

**Pull request merged**